docs/experiments/focus-metavariable.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: focus-metavariable
+append_help_link: true
+description: "focus-metavariable allows Semgrep users to narrow down the code matched by a rule, to focus on what it really matters."
+---
+
+# `focus-metavariable`
+
+The experimental `focus-metavariable` operator puts the focus, or "zooms in", on the code region matched by a metavariable. For example, if we wanted to find all functions arguments annotated with the type `bad` we could write this pattern:
+
+```yaml
+pattern: |
+  def $FUNC(..., $ARG : bad, ...):
+    ...
+```
+
+This works but it matches the entire function definition. Sometimes that is not desirable, for instance, if we are using [Semgrep App](https://semgrep.dev/login) and we have triaged a finding generated by this pattern, the same finding may show up again as new if we make any change to the definition of the function!
```

Comment:
```suggestion
This works but it matches the entire function definition. Sometimes, this is not desirable, for example, if you are using [Semgrep App](https://semgrep.dev/login) and you have triaged a finding generated by this pattern, the same finding may show up again as new if you make any change to the definition of the function!
```

Reply:
Check the use of [_we_ section in our style guide](https://www.notion.so/r2cdev/r2c-technical-communication-style-guide-4d1104e78927460aa8c7624aedd6561f#39871bd6aca04e37bac6dc2e70dd2baf). :-)